runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -89,6 +98,10 @@
 	private LootTrackerPanel panel;
 	private NavigationButton navButton;
 	private String eventType;
+	private int lastNumberOfSupplyCrates;
+	private HashMultiset<Integer> inventorySnapshot;
+	private static final String WINTERTODT_EVENT_NAME = "Wintertodt";
+	private static final Collection SUPPLY_CRATES = ImmutableList.of(ItemID.SUPPLY_CRATE, ItemID.EXTRA_SUPPLY_CRATE);
```

Comment:
I am not sure if you should track EXTRA_SUPPLY_CRATE, as that one is basically buyable from GE and not unique from WT